libsolutil/Whiskers.cpp

```diff
@@ -100,6 +108,19 @@ void Whiskers::checkParameterUnknown(string const& _parameter) const
 	);
 }
 
+void Whiskers::checkTemplateContainsTags(string const& _parameter, vector<string> const& _tags)
+{
+	for (auto const& aTag: _tags)
+	{
+		string tag{"<" + aTag};
```

Comment:
```suggestion
		string tag{"<" + aTag + _parameter + ">"};
```
I find it more readable this way